cms/articles/utils.py

```diff
@@ -0,0 +1,34 @@
+from typing import TYPE_CHECKING
+
+from django.utils.formats import date_format
+from django.utils.translation import gettext_lazy as _
+
+if TYPE_CHECKING:
+    from wagtail.blocks.stream_block import StreamChild
+
+
+def serialize_correction_or_notice(entry: "StreamChild", *, superseded_url: str | None = None) -> dict:
+    """Serialize a correction or notice entry for the API.
+
+    Args:
+        entry (StreamChild): The stream child containing the correction or notice data.
+        superseded_url (str | None): Optional URL to the superseded version of the article
+            if this is a correction. If provided, the item will be labeled as a correction.
+
+    Returns:
+        dict: A dictionary containing the serialized data.
+    """
+    content = {
+        "text": _("Correction") if superseded_url else _("Notice"),
+        "date": {
+            "iso": date_format(entry.value["when"], "c"),
+            "short": date_format(entry.value["when"], "DATE_FORMAT"),
```

Comment:
this is not quite right. Corrections need to have the time too (Date, time)

![Screenshot 2025-06-30 at 11 43 39](https://github.com/user-attachments/assets/3c1da361-42c1-4135-9c40-0aa1f854c4cc)

Reply:
Fixed in [164a0ba](https://github.com/ONSdigital/dis-wagtail/pull/260/commits/164a0bada18e0ebddb1dd4f70cd8106db3e3a560)